workers/loc.api/sync/currency.converter/index.js

```diff
@@ -159,9 +159,39 @@ class CurrencyConverter {
       return
     }
 
-    return synonymous.map(([symbol, conversion]) => (
-      [`${prefix}${symbol}${lastSymb}`, conversion]
-    ))
+    return synonymous.reduce((accum, [symbol, conversion]) => {
+      const separator = (
+        symbol.length > 3 ||
+        lastSymb.length > 3
+      )
+        ? ':'
+        : ''
+
+      accum.push([
+        `${prefix}${symbol}${separator}${lastSymb}`,
+        conversion
+      ])
+
+      if (
```

Comment:
in this case wont it push to accum two times? 
Wont be better to have an object with all pairs and then search there? 
allPairs[symbol][lastSymb]
I think this would make searches pretty fast also.